.github/workflows/test.yaml

```diff
@@ -31,48 +37,48 @@ jobs:
           - dev
           - release
 
-        target_triple:
-          - name: x86_64-pc-windows-msvc
-            arch: amd64
-          # - aarch64-pc-windows-msvc FIXME: Add support for executing ARM64 tests. Maybe use target specific test runner in emulator: https://doc.rust-lang.org/cargo/reference/config.html#target
+        # Exclude WDK 10.0.22621 with ARM64 runner as it doesn't ship certmgr and makecert binaries
+        exclude:
+          - runner:
+              name: windows-11-arm
+              arch: arm64
+            wdk:
+              version: 10.0.22621
+              source: winget
+
+    runs-on: ${{ matrix.runner.name }}
 
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v4
 
       - name: Install Winget
-        uses: ./.github/actions/winget-install
+        uses: ./.github/actions/install-winget
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Install LLVM ${{ matrix.llvm }}
-        run: |
-          if ((Get-WinGetPackage -Id LLVM -Source winget -MatchOption Equals).InstalledVersion -eq '${{ matrix.llvm }}') {
-            Write-Host "LLVM ${{ matrix.llvm }} is already installed."
-          } else {
-            Write-Host "Installing LLVM ${{ matrix.llvm }}..."
-            Install-WinGetPackage -Id LLVM.LLVM -Version ${{ matrix.llvm }} -Source winget -MatchOption Equals -Mode Silent -Force
-          }
-          clang --version
+        uses: ./.github/actions/install-llvm
+        with:
+          version: ${{ matrix.llvm }}
 
       - name: Install WDK (${{ matrix.wdk.version }})
         uses: ./.github/actions/install-wdk
         with:
           version: ${{ matrix.wdk.version }}
           source: ${{ matrix.wdk.source }}
-          host: 'x64' # windows-latest is x64-based
-          target: ${{ matrix.target_triple.arch }}
+          host: ${{ matrix.runner.arch }}
+          target: ${{ matrix.runner.arch }}
 
       - name: Install Rust Toolchain (${{ matrix.rust_toolchain }})
         uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{ matrix.rust_toolchain }}
-          targets: ${{ matrix.target_triple.name }}
 
       - name: Install Cargo Expand
         uses: taiki-e/install-action@v2
         with:
           tool: cargo-expand@1.0.85
 
       - name: Run Cargo Test
-        run: cargo +${{ matrix.rust_toolchain }} test --locked --profile ${{ matrix.cargo_profile }} --target ${{ matrix.target_triple.name }} --all-features
```

Comment:
With this, we are losing the check that tests cross compile successfully

Reply:
Please refer to my earlier discussion here https://github.com/microsoft/windows-drivers-rs/pull/420#discussion_r2257156850

Quoting it here
>For test workflow, cross compilation wouldn't make sense as unit tests are generally built as exes to the target architecture (assuming target is different from host), and those binaries can't be run on the host architecture as part of test run. However, there is a provision to make these unit tests run by passing a custom runner configuration in the config.toml matching the target architecture. I believe it is not necessary to explore the custom runner configuration for this PR scope and left it out.

Is there anything else we could do here?

Reply:
1. If I understand correctly, you're referring to using runner configuration for executing cross-compiled tests? I think we can create a separate issue for this if that makes sense.
2. I have one question. For example, if we cross-compile tests to AArch64 on an x64 machine, what additional benefit do we gain if we're already running the same tests on an ARM64 runner? What does cross-compiling the tests actually provide?
3. Just for clarity, the way I see it, the existing matrix has only covered host compilation so far. While it includes a placeholder for cross-compilation, but it was not implemented. So, we're not reducing the current coverage; instead, we're extending it to include test runs on ARM64 as well.

Reply:
1. No. i would just cross compile the arm64 tests, and copy them over to an arm64 host manually. 
2. only small benefit is a check that the cross-compilation works
3. ah. ur right, it was commented out

anyways, i think this is fine